pkg/utils/options/options.go

```diff
@@ -45,6 +45,7 @@ func MustParse() Options {
 	flag.BoolVar(&opts.AWSENILimitedPodDensity, "aws-eni-limited-pod-density", env.WithDefaultBool("AWS_ENI_LIMITED_POD_DENSITY", true), "Indicates whether new nodes should use ENI-based pod density")
 	flag.StringVar(&opts.AWSDefaultInstanceProfile, "aws-default-instance-profile", env.WithDefaultString("AWS_DEFAULT_INSTANCE_PROFILE", ""), "The default instance profile to use when provisioning nodes in AWS")
 	flag.BoolVar(&opts.AWSEnablePodENI, "aws-enable-pod-eni", env.WithDefaultBool("AWS_ENABLE_POD_ENI", false), "If true then instances that support pod ENI will report a vpc.amazonaws.com/pod-eni resource")
+	flag.IntVar(&opts.AWSVolumeAttachLimit, "aws-volume-attach-limit", env.WithDefaultInt("AWS_VOLUME_ATTACH_LIMIT", -1), "If set to a value >=0, this is the maximum number of volumes attachable per node and should match the value passed to the EBS CSI driver.")
```

Comment:
Worth linking to https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/92feaef772b7c93c7da3a5b282a1f9bac819a319/docs/options.md?